pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -382,6 +384,9 @@ message PodSandboxStatus {
 message PodSandboxStatusResponse {
     // Status of the PodSandbox.
     PodSandboxStatus status = 1;
+    // Info is extra information of the PodSandbox in json format.
+    // It should only be returned non-empty when Verbose is true.
+    string info = 2;
```

Comment:
Maybe a `map[string]string` would be a better fit for this? How are you expecting to consume the info?

Reply:
@tallclair if the content is json, `map[string]string` or string are similar. A `map[string]string` could be marshaled to one string as well.

Reply:
I think it's a valid question. Arbitrary JSON format gives you freedom to express hierarchies, which the flattened key-value map doesn't have. On the other hand, the `map[string]string` field makes the information more consumable directly, without the additional tooling to marshal/unmarshal the string.

Do we think a single-level key-value map is sufficient to express the debug information?

Reply:
> Do we think a single-level key-value map is sufficient to express the debug information?

I don't think so. The value may still need to be a json, even if we use `map[string]string`. For example, if we want to include container config or runtime spec inside the debug information, the key will be "config" and the value will be the spec, which is another json string.

On the container runtime side, there will be a struct `ContainerInfo`, e.g.
```go
type ContainerInfo {
  Spec *runtime.Spec
  Pid uint32
}
```
It could be marshaled into `string`:
```json
{
  "config": "... json runtime spec ...",
  "pid": "123"
}
```
It could also be marshal into string and unmarshaled into `map[string]string`:
```go
map[string]string = {
"config": "... json runtime spec ...",
"pid": "123"
}
```
I'm fine with either. :) `map[string]string` might be a bit more flexible.

Reply:
I prefer the latter for a few reasons:
- Easier to work with for implementers
- Easier to format in a human-readable way for `crictl`
- Consistent with kubernetes APIs
- More flexible (e.g. 2 different components want to add config)

Reply:
Done